pkg/cmd/scan/output/html.go

```diff
@@ -96,6 +96,17 @@ func (c *HTML) Write(analysis *analyser.Analysis) error {
 
 			return distinctResourceTypes(resources)
 		},
+		"getIaCSources": func() []string {
+			resources := make([]*resource.Resource, 0)
+			resources = append(resources, analysis.Deleted()...)
+			resources = append(resources, analysis.Managed()...)
+
+			for _, d := range analysis.Differences() {
```

Comment:
This loop is useless since diff always happen on managed resources

Reply:
right, thanks